YvesIraguha/TeaHouse
# #169975449 Enable Paginations on Collections

#### What does this PR do?
- Enable paginations and filtering on collections
#### Description of Task to be completed?
- Paginate the output of fetching all collections
- Filter the output based on book series or images
#### How should this be manually tested?
- Clone the repo and checkout the branch `paginations-collections-169975449`
- Run `npm i && npm run precommit && npm run start` to run tests and start the server
- Consume the endpoint `/api/v1/collections?page=number&type=Book series`, and you should see the paginated outcome.

#### What are the relevant pivotal tracker stories?
#[169975449](https://www.pivotaltracker.com/story/show/169975449)
